pkgs/by-name/me/meson/package.nix

```diff
@@ -93,6 +95,11 @@ python3.pkgs.buildPythonApplication rec {
     OpenAL
     OpenGL
     openldap
+  ] ++ lib.optionals (stdenv.cc.isClang && !stdenv.isDarwin) [
+    # https://github.com/mesonbuild/meson/blob/bd3f1b2/test%20cases/common/184%20openmp/meson.build
```

Comment:
We should use the full hash in long term links.

Reply:
done